Tweetinvi.Controllers/Properties/Resources.cs

```diff
@@ -455,10 +460,10 @@ internal static class Resources
         public static string Tweet_PublishWithMediaInReplyTo = "https://api.twitter.com/1.1/statuses/update_with_media.json?status={0}&in_reply_to_status_id={1}";
 
         /// <summary>
-        ///   Looks up a localized string similar to https://api.twitter.com/1.1/statuses/retweets/{0}.json.
+        ///   Looks up a localized string similar to https://api.twitter.com/1.1/statuses/retweets/{0}.json?count={1}.
         /// </summary>
-        public static string Tweet_Retweet_GetRetweets = "https://api.twitter.com/1.1/statuses/retweets/{0}.json";
-        
+        public static string Tweet_Retweet_GetRetweets = "https://api.twitter.com/1.1/statuses/retweets/{0}.json?count={1}";

```

Comment:
As mentioned below, the resources string should no longer have parameters added directly. Instead it should be `https://api.twitter.com/1.1/statuses/retweets/{0}.json` and the `QueryGenerator` will use `.AddQueryParameter` to add the count parameter to the query.